gprofiler/profilers/java.py

```diff
@@ -330,22 +330,22 @@ def get_jvm_flags(self, process: Process) -> Optional[Iterable[JvmFlag]]:
         if self.java_collect_jvm_flags == JavaFlagCollectionOptions.NONE:
             return None
 
-        jvm_raw_flags: Iterable[JvmFlag] = filter(self.filter_jvm_flag, self.get_jvm_flags_raw(process))
+        filtered_jvm_flags = self.get_supported_jvm_flags(process)
```

Comment:
supported_jvm_flags?

Reply:
supported for collection, `basic_filter` sounds unclear IMO

Reply:
Ok